sisl/viz/plots/fatbands.py

```diff
@@ -186,6 +186,10 @@ class FatbandsPlot(BandsPlot):
 
     )
 
+    @property
+    def weights(self):
+        return self.bands_data["weight"]
```

Comment:
perhaps it should also be named `weights` in the data? for consistency?

Reply:
I'm not sure, because for me the logic way to call the variable is `weight`. It feels strange to give a variable a plural name, doesn't it?

In fact I was thinking quite the opposite: that the `spin_moments` variable should be renamed to `spin_moment`.

Reply:
I don't really care ;) but you are using `weights` and `weight` here?
I tend to agree that plural is not aiding any understanding and hence I also try to avoid plural s

Reply:
`weight` is the name of the variable in the dataset, `weights` is what feels natural to me as an attribute if I want all the weights of the orbitals.

Reply:
ok, fine.